src/declarations.d.tsx

```diff
@@ -3,3 +3,16 @@
 declare type Observable<T> = import("rxjs").Observable<T>;
 
 declare module "*.css";
+
+type Referral = {
```

Comment:
This was just moved from another file into this "common declarations" file so that it can be used anywhere.

The reason it's helpful to have this defined is that it allows Typescript to check references to the data, and for VSCode to autocomplete.